velox/exec/fuzzer/PrestoQueryRunner.h

```diff
@@ -74,6 +80,16 @@ class PrestoQueryRunner : public velox::exec::test::ReferenceQueryRunner {
       const std::vector<RowVectorPtr>& input,
       const RowTypePtr& resultType) override;
 
+  std::vector<RowVectorPtr> executeVector(
+      const std::string& sql,
+      const std::vector<RowVectorPtr>& probeInput,
+      const std::vector<RowVectorPtr>& buildInput,
+      const RowTypePtr& resultType) override;
+
+  std::string createTable(
+      const std::vector<velox::RowVectorPtr>& input,
+      const std::string& tableName) override;
```

Comment:
Let's move it to a private function and remove override

Reply:
Also can you reorder the parameters to be the same as 
```
const std::string& name, 
const std::vector<RowVectorPtr>& data
```

https://github.com/facebookincubator/velox/blob/ff07cb0b757050c2a631ede72f7f6cb0586c4d47/velox/exec/tests/utils/QueryAssertions.cpp#L893-L895